src/lib/constants.py

```
@@ -15,7 +15,7 @@
     "claude-4.5-sonnet",
     "claude-4.5-opus",
     "claude-4.1-opus",
-    "claude-4.5-haiku",
+    #"claude-4.5-haiku", - does not support JSON output
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)